### DISCUSSION TITLE
Unable to Register using Github account, unable to login using e-mail-phone verification code not sent/received

### DISCUSSION THREAD

**Discussion by lvega77** [vercel/community#4428] [Account Issue] [answered]
### Question

Good evening:

I've attempted to register to Vercel using my GitHub account.  I ensured my e-mails in GitHub are visible to the public after reading this as a possible solution in a different blog, however, my Github account appears to be blocked by Vercel.  I've never used Vercel before and I'm unsure why would they block my Github account even prior of me attempting to register.

Due to this, I attempted to register using multiple e-mails.  I used my student e-mail, I used my g-mail, and I used my outlook email.  Each time Vercel directed me to a "one time code verification" page, in which I entered my cellphone number and I was supposed to received an SMS message with a code.  I can't tell how many times I've tried this and never received the message.

I've e-mailed Vercel support once from each e-mail that I've attempted to register with, as far back as 21 September 2023, and still haven't received any response other than the automated e-mail with a reference number.

I'm attempting to use Vercel to deploy codes as part as a University course, I have no need to upgrade to anything more than a Hobby account. I'm running out of ideas since Vercel hasn't contacted me, and there seems to be no direct way to contact their support staff.

**Comment by swarnava** [comment, marked as answer]
Please email us at registration@vercel.com
